lib/highlight-line.coffee

```diff
@@ -19,33 +20,52 @@ module.exports =
       default: 'solid'
       enum: ['solid', 'dotted', 'dashed']
   line: null
+  subscriptions: null
 
   activate: ->
     @line = new HighlightLineView()
     @line.attach()
 
-    atom.workspaceView.command(
-      'highlight-line:toggle-background', '.editor', =>
-        @toggleHighlight()
-    )
-    atom.workspaceView.command(
-      'highlight-line:toggle-hide-highlight-on-select', '.editor', =>
-        @toggleHideHighlightOnSelect()
-    )
-    atom.workspaceView.command(
-      'highlight-line:toggle-underline', '.editor', =>
-        @toggleUnderline()
-    )
-    atom.workspaceView.command(
-      'highlight-line:toggle-selection-borders', '.editor', =>
-        @toggleSelectionBorders()
-    )
+    # Setup to use the new composite disposables API for registering commands
+    @subscriptions = new CompositeDisposable
+
+    # Add the commands
+    @subscriptions.add atom.commands.add "atom-workspace",
+        'highlight-line:toggle-background': => @toggleHighlight()
+    @subscriptions.add atom.commands.add "atom-workspace",
+        'highlight-line:toggle-hide-highlight-on-select': => @toggleHideHighlightOnSelect()
```

Comment:
The build is failing because this line is too long.

    @subscriptions.add atom.commands.add "atom-workspace",
        'highlight-line:toggle-hide-highlight-on-select': => 
            @toggleHideHighlightOnSelect()